source/install/install-kubernetes-mattermost.rst

```diff
@@ -80,30 +188,158 @@ The Mattermost installation manifest contains fields which must be edited in lin
     "spec.elasticSearch.username", "The username for your Elasticsearch instance.", "No"
     "spec.elasticSearch.password", "The password for your Elasticsearch instance.", "No"
 
-There are more advanced fields documented `here <https://raw.githubusercontent.com/mattermost/mattermost-operator/master/docs/examples/full.yaml>`__.
+7. Create a restore manifest:
 
-**4. Apply the Installation Manifest File**
+Open a text editor and create a text file with the following details. Save the file as ``restore.yaml``:
 
-To initiate deployment, apply the file, specifying the correct path, using:
+.. code-block:: yaml
+
+  apiVersion: mattermost.com/v1alpha1
+  kind: MattermostRestoreDB
+  metadata:
+    name: example-mattermostrestoredb
+  spec:
+    initBucketURL: s3://my-sample/my-backup.gz
+    mattermostClusterName: example-clusterinstallation
+    mattermostDBName: mattermostdb
+    mattermostDBPassword: supersecure
+    mattermostDBUser: mmuser
+    restoreSecret: myawscreds
+
+**Parameters**
+- ``mattermostClusterName``. The ClusterInstallation file name.
+- ``RestoreSecret``. The location of the backup file.
+- ``mattermostDBPassword``. The password used to access the database.
+- ``mattermostDBUser``. The username required to access the database.
+- ``initBucketURL``. The URL of the storage instance/server where the backed up DB is stored.
+
+8. To initiate deployment, apply the file and specify the path where the newly-created files have been saved:
 
 .. code-block:: sh
 
-  $ kubectl create ns mattermost
-  $ kubectl apply -n mattermost -f /path/to/mattermost-installation.yaml
+      $ kubectl create ns mattermost
+      $ kubectl apply -n mattermost -f /path/to/mattermost-installation.yaml
 
-The deployment process can be monitored in the Kubernetes user interface.
+The deployment process can be monitored in the Kubernetes user interface. If errors or issues are experienced,
+review the Mattermost log files or MySQL logs for guidance including error messages. If remediation is not successful, contact
+Mattermost customer support for assistance.
 
-**4. Configure DNS and Use Mattermost**
+Once complete, access your Mattermost instance and confirm that the database has been restored.
 
-When the deployment is complete, obtain the hostname or IP address of your Mattermost deployment using the following command:
+Using an External Database with the Mattermost Operator
```

Comment:
This section is a copy of the restore DB section and review is needed to confirm content.

Reply:
@jwilander, I thought this section would simply be a replication of the restore DB section but it needs a review as I don't think my understanding is accurate.

Reply:
This whole case doesn't need to exist. If the database is external than the operator doesn't need to manage it at all

Reply:
It is duplicate content and I totally understand why it doesn't need to be in the docs. My next question is how do we address the customer questions around using an existing db with MM operator (which is why I added this). It seemed like these would be the correct steps minus the references to restore files and replacing them with something like "newDB" but it didn't turn out as expected.